polyhost/res/overlay_sources/rect_mark.py

```
@@ -0,0 +1,142 @@
+        if Path(p).exists():
+            return ImageFont.truetype(p, size)
+    raise RuntimeError(f"no bold face found; looked in {fam}")
+
+
+PAD = BORDER + 2
+
+
+def _fits(d: ImageDraw.ImageDraw, letters: str, size: int, serif: bool,
+          avail_w: int, avail_h: int):
+    f = _font(size, serif)
+    l, t, r, b = d.textbbox((0, 0), letters, font=f)
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)